running locally/docker-prestissimo/prestissimo/coordinator/log.properties

```diff
@@ -0,0 +1 @@
+com.facebook.presto=DEBUG
```

Comment:
please make sure there is a newline at the end of all files.

Reply:
Done